src/Main.java

```diff
@@ -1,3 +1,5 @@
+import hangedMan.logic.Game;
+
 public class Main {
```

Comment:
This class still uses default package. The issue was, that all classses should be in named package

Reply:
That is correct, IDEA couldn't somehow manage to find the Main, when it wasn't in the default. I think I tried putting it into hangedMan.logic when the problem ocurred. Main should be in the correct package in following commit.